doc/ref_model/chapters/chapter02.md

```diff
@@ -266,17 +266,44 @@ By trying to sort workloads into different categories based on the requirements
 <a name="2.4"></a>
 ## 2.4 Cloud Infrastructure Profiles
 
-Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
+Infrastructure profiles are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities exploitable by the workloads.
+Two profile layers are proposed:
+
+- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, ie: an infrastructure object can belong to one and only one profile, and workloads can request to be instantiated on one infrastructure profile only.
+- Within each profile, **flavours** represent small deviations from the profile baseline that do not require partitioning the infrastructure into separate pools, whose specifications have a finer granularity than top-level profiles, and can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, ie: an infrastructure object can have **more than one flavour** attached to it, and workloads can request to be instantiated infrastructure with a number of (0+) certain flavours.
```

Comment:
too many new term "profile baseline" "top-level profiles"

```suggestion
- For a given profile, **flavours** represent small deviations from or further qualification (such as sizing) of the profile that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Flavours can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, ie: an infrastructure resource can have **more than one flavour** attached to it, and workloads can request VMs to be instantiated on infrastructure with a certain flavour.
```

Reply:
thanks, have added this